README.md

```diff
@@ -93,16 +111,20 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
-```text
-
+```
+You call a function that you want executed now by writing it as someVariable = someFunction (). You reference a function that you want to run upon some other action preceding it by writing it as someVariable = someFunction. This would imply that someVariable is an alias for someFunction. If we call someVariable(), we would get the same result as someFunction().
 ```
```

Comment:
:+1:  just a small note, we don't always have to save the output of a function call. ex: 
```js
  function doSomething(){
    console.log("Something's happening")
}

doSomething()
``` 
will execute the `doSomething` function
```